File: examples/4/odd0.py
#!/usr/local/bin/python3.5 -u
import sys

number = 0
while number >= 0:
    print("Enter number:")
    number = float(sys.stdin.readline())
    if number >= 0:
        if number % 2 == 0:
            print("Even")
        else:
            print("Odd")
print("Bye")

File: examples/4/odd0.pl
#!/usr/bin/perl -w

$number = 0;
while ($number >= 0) {
    print "Enter number:\n";
    $number = <STDIN>;
    if ($number >= 0) {
        if ($number % 2 == 0) {
            print "Even\n";
        } else {
            print "Odd\n";
        }
    }
}
print "Bye\n";
